tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_SUBCONUS_Ind_3km_ics_RAP_lbcs_RAP_suite_RRFS_v1beta.yaml

```diff
@@ -8,18 +8,20 @@ user:
 workflow:
```

Comment:
Should the description note the change in ICs being derived from RAP rather than HRRR?